content/neuvector-prime/azure/contents.lr

```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+To Update the NeuVector version to the latest version supported by the marketplace listing, via the "az k8s-extension update" command.
+
+Execute the following commands in a Cluster Cloud Shell:
```

Comment:
this section above needs some more work...  the "To Update..." sentence basically says "to update do" but never tells what to do... Maybe:
"Updating the NeuVector version to the latest version supported by the marketplace listing can be accomplished using the "az k8s-extension update" command.

then update the Execute line to something like:
"The following az k8s-extension commands should be executed in a Cluster Cloud Shell."

Tweak the note to read "NOTE the list-versions command to show the latest published version is available in the westcentralus region and will be available in other regions as the feature is rolled out to those regions".